--- guvcview-src-2.0.2/gview_v4l2core/uvc_h264.c~	2015-07-27 23:26:48.000000000 +0900
+++ guvcview-src-2.0.2/gview_v4l2core/uvc_h264.c	2016-03-31 20:35:23.109698271 +0900
@@ -970,7 +970,7 @@
 	}
 	
 	h264_ctx->context->flags2 |= CODEC_FLAG2_FAST;
-	h264_ctx->context->pix_fmt = PIX_FMT_YUV420P;
+	h264_ctx->context->pix_fmt = AV_PIX_FMT_YUV420P;
 	h264_ctx->context->width = width;
 	h264_ctx->context->height = height;
 	//h264_ctx->context->dsp_mask = (FF_MM_MMX | FF_MM_MMXEXT | FF_MM_SSE);
--- guvcview-src-2.0.2/gview_v4l2core/jpeg_decoder.c~	2015-07-27 23:26:48.000000000 +0900
+++ guvcview-src-2.0.2/gview_v4l2core/jpeg_decoder.c	2016-03-31 20:35:56.186940070 +0900
@@ -1436,7 +1436,7 @@
 		exit(-1);
 	}
 
-	codec_data->context->pix_fmt = PIX_FMT_YUV422P;
+	codec_data->context->pix_fmt = AV_PIX_FMT_YUV422P;
 	codec_data->context->width = width;
 	codec_data->context->height = height;
 	//jpeg_ctx->context->dsp_mask = (FF_MM_MMX | FF_MM_MMXEXT | FF_MM_SSE);
